ArchiSteamFarm/www/js/app.js

```diff
@@ -458,6 +458,15 @@ function prepareConfigForSaving() {
                         break;
                     }
 
+                    if ($keyValue !== value) config[key] = $keyValue;
+                    break;
+					
+				case 'System.Net.WebProxy':
```

Comment:
What do we need it for? ASF does not expose `System.Net.WebProxy` anywhere in API IIRC.

Reply:
`/Api/Type/ArchiSteamFarm.GlobalConfig` does return `System.Net.WebProxy`.

![webproxy](https://user-images.githubusercontent.com/31552675/42833001-46781f7e-89f3-11e8-8e58-77b2c9eee06c.PNG)

Reply:
Interesting, indeed ASF under the hood uses that object for its own operation but IPC should return that one as string, I might want to correct this.

Reply:
Okay, let me know when IPC returns `WebProxy` as string, then I will revert the changes regarding it.

Reply:
Done, it's string now, as it should be in the first place.